c-bindings/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
 go_library(
     name = "go_default_library",
     srcs = [
+        "handles.go",
```

Comment:
Hmmm... I wonder how it was passing tests before without that... Maybe the bazel check is not building this part. We should investigate this (ofc, out of scope for this change).